docs/reference/cheat-sheet.md

```diff
@@ -0,0 +1,452 @@
+---
+description: flagd cheat sheet - quick reference for common operations
+---
+
+# Cheat sheet
+
+This cheat sheet provides quick reference examples for running flagd and evaluating flags using various protocols and configurations.
+
+Recommended tools:
+
+- [docker](https://docs.docker.com/)
+- [curl](https://curl.se/)
+- [grpcurl](https://github.com/fullstorydev/grpcurl)
+- [jq](https://jqlang.org/) (optional, for formatting)
+
+!!! tip
+
+    These commands assume a unix-like shell.
+
+    Output is generally JSON, and can be pretty-printed by piping into `jq` (ie: `curl ... | jq`)
+
+## Sample Flag Definitions
+
+The examples below use these sample flag definition files. Download them to follow along:
+
+- [cheat-sheet-flags.json](../assets/cheat-sheet-flags.json) - General application flags (flagSetId: `app-flags`)
+- [cheat-sheet-flags-payments.json](../assets/cheat-sheet-flags-payments.json) - Payment-related flags (flagSetId: `payment-flags`)
+
+The `app-flags` set includes:
+
+| Flag Key | Type | Description |
+|----------|------|-------------|
+| `simple-boolean` | boolean | Static boolean flag |
+| `simple-string` | string | Static string flag |
+| `simple-number` | integer | Static numeric flag |
+| `simple-object` | object | Static object flag |
+| `user-tier-flag` | string | Context-sensitive flag based on `tier` |
+| `email-based-feature` | boolean | Context-sensitive flag based on `email` domain |
+| `region-config` | object | Context-sensitive flag based on `region` |
+
+The `payment-flags` set includes:
+
+| Flag Key | Type | Description |
+|----------|------|-------------|
+| `payment-provider` | string | Static payment provider selection |
+| `max-transaction-amount` | integer | Context-sensitive based on `account-verified` |
+| `enable-crypto-payments` | boolean | Context-sensitive based on `country` |
+
+---
+
+## Running flagd
+
+=== "Binary"
+
+    ```shell
+    # Single flag source (local file)
+    flagd start --uri file:./cheat-sheet-flags.json
+
+    # Multiple flag sources
+    flagd start \
+      --uri file:./cheat-sheet-flags.json \
+      --uri file:./cheat-sheet-flags-payments.json
+
+    # HTTP source
+    flagd start --uri https://flagd.dev/assets/cheat-sheet-flags.json
+    ```
+
+=== "Docker"
```

Comment:
I'd recommend putting Docker as the first option as it's the most straightforward way to run flagd.